pkg/cloudprovider/providers/gce/gce_alpha.go

```diff
@@ -29,15 +29,12 @@ const (
 	// tier to use. Currently supports "Standard" and "Premium" (default).
 	AlphaFeatureNetworkTiers = "NetworkTiers"
 
-	AlphaFeatureGCEDisk = "DiskAlphaAPI"
-
```

Comment:
Double check removing this feature does not cause any failures if old code still passes the flag in.

Reply:
This is the only spot that passes in an alpha features list that could contain "DiskAlphaAPI", and it just logs the error if the feature doesn't exist.

https://github.com/kubernetes/kubernetes/blob/6a7656b69364337749133f97d7dcb78f63f36b60/pkg/cloudprovider/providers/gce/gce.go#L294